doc/nrf/device_guides/nrf54l.rst

```diff
@@ -0,0 +1,24 @@
+.. _ug_nrf54l:
+
+Working with nRF54L Series
+##########################
+
+Zephyr and the |NCS| provide support and contain board definitions for developing on the following nRF54L Series devices:
+
+.. list-table::
+   :header-rows: 1
+
+   * - DK
+     - PCA number
+     - Build target
+     - Documentation
+   * - :ref:`zephyr:nrf54l15pdk_nrf54l15`
+     - PCA10156
+     - ``nrf54l15pdk_nrf54l15_cpuapp``
```

Comment:
It's only soc1 that's supported now AFAIK, so I think we might want to change this from nrf54l15pdk_nrf54l15_cpuapp to nrf54l15dk_nrf54l15_cpuapp@soc1

Reply:
updated